README.md

```diff
@@ -375,31 +363,35 @@ Please execute at least step 1 + 2 if before importing restdocs-wiremock into yo
 
 Given that the `master` branch on the upstream repository is in the state from which you want to create a release, execute the following steps:
 
-```shell
-git checkout master
-git pull upstream master
+1. Create release via the GitHub UI
 
-# Print current version to the terminal
-./gradlew currentVersion
+https://github.com/ePages-de/restdocs-wiremock/releases/new
 
-# (a) Release new patch version
-./gradlew release -Prelease.versionIncrementer=incrementPatch
+Use the intended version number as "Tag version", e.g. "0.8.0".
 
-# (b) Release new minor version
-./gradlew release -Prelease.versionIncrementer=incrementMajor
+This will automatically trigger a Travis build which publishes the JAR files for this release to Sonatype.
 
-## (c) Release new major version
-./gradlew release -Prelease.versionIncrementer=incrementMajor
+2. Login to Sonatype
 
-# Print current version to the terminal
-./gradlew currentVersion
-```
+Login to Sonatype and navigate to the [staging repositories](https://oss.sonatype.org/#stagingRepositories).
+
+3. Close the staging repository
+
+Select the generated staging repository and close it.
+Check that there are no errors afterwards (e.g. missing signatures or Javadoc JARs).
+
+4. Release the repository
+
+Select the genereted staging repository and publish it.
+Soon after, the release should be available in the in the "Public Repositories" of ePages:
+
+https://oss.sonatype.org/service/local/repo_groups/public/content/com/epages/restdocs-wiremock/maven-metadata.xml
+
+(Steps 2 to 4 could be automated, if we want to. See [publish-plugin/README.md](https://github.com/gradle-nexus/publish-plugin/blob/master/README.md#publishing-to-maven-central-via-sonatype-ossrh))
 
-TravisCI will then take care to call the Gradle tasks which upload the release to Sonatype.
-A new staging repository will be create at [oss.sonatype.org](https://oss.sonatype.org/#stagingRepositories).
-You need to login there, go to the latest staging repository, and then close and release the repository in the Sonatype UI.
+5. Update documentation
 
-(Once this process is working reliably we can also automate the final manual steps.)
+Create a new commit which updates the version numbers in the `README` file.
```

Comment:
What would you think about having this as a first step of the publishing procedure? It then would be part of the release.

Reply:
I think we should not publicly document how to use a version which doesn't exist.

Doing it as suggested would create some time in which users could try to follow the README and it doesn't work. In the minium case this is the time it takes to create the build and publish it. In the average case it is some more time because of the sync between the reviewer, the PR author, and the one who creates the release. In the worst case the "downtime" is a few hours or days, if something goes wrong in the publication process.

Further, before I publicly document how to use a library, I would like to try this out by myself, first. This I cannot do before the library is actually released.

Reply:
I see your point of view, too. Both have their advantages and disadvantages. I'm fine to create the tag afterwards.